service/view_building_coordinator.cc

```diff
@@ -184,18 +185,25 @@ future<std::optional<std::pair<group0_guard, view_building_coordinator::vbc_stat
 
     // Since vbc_state is immutable, select new base table to process only if there are no other changes to the state.
     if (!state.currently_processed_base_table && !state.tasks.empty() && cmuts.empty()) {
+        SCYLLA_ASSERT(state.targets_with_staging_sstables.empty());
+        
         auto& base_id = state.tasks.cbegin()->first;
         vbc_logger.info("Start building views for base table: {}", base_id);
 
         auto mut = co_await _sys_ks.make_vbc_processing_base_mutation(guard.write_timestamp(), base_id);
         cmuts.emplace_back(std::move(mut));
+        notify_others_to_detect_staging = true;
     }
 
     if (!cmuts.empty()) {
         auto cmd = _group0.client().prepare_command(write_mutations{
             .mutations{std::move(cmuts)},
         }, guard, "update view building coordinator state");
         co_await _group0.client().add_entry(std::move(cmd), std::move(guard), _as);
+
+        if (notify_others_to_detect_staging) {
+            co_await notify_others_to_detect_staging_sstables();
+        }
```

Comment:
What if you manage to commit the mutation which sets the `vbc_processing_base` key in system.scylla_local, but then this call fails (e.g. there is a temporary connection failure to one of the nodes)? Will the view build coordinator retry the operation?

Reply:
removed this RPC, not each node checks locally if processing base table was change and does initial run of detecting staging sstables on the new base table